models/addresses/ethereum/addresses_ethereum_l2_batch_submitters.sql

```diff
@@ -4,29 +4,45 @@
                                     "addresses",
                                     \'["msilb7"]\') }}') }}
 
-SELECT lower(address) as address, protocol_name, submitter_type, role_type, version, description
+SELECT address, protocol_name, submitter_type, role_type, version, description
```

Comment:
Why is the lower() here removed?

Reply:
manually forced all entries to "lower" and it seems like having lower forces the entries to be varchar instead of varbinary in dunesql

Reply:
i'll readd it just for safety